component/attr_cache/attr_cache.go

```diff
@@ -638,7 +599,12 @@ func (ac *AttrCache) DeleteFile(options internal.DeleteFileOptions) error {
 		deletionTime := time.Now()
 		ac.cacheLock.RLock()
 		defer ac.cacheLock.RUnlock()
-		ac.deletePath(options.Name, deletionTime)
+		toBeDeleted, err := ac.cacheMap.get(options.Name)
+		if err != nil {
+			log.Err("cannot find the attr cache item due to the following error: ", err)
```

Comment:
```suggestion
			log.Err("AttrCache::DeleteFile : %s", err)
```
Error style,
Shorten the error to something simple and specific (the error that `get()` returns is `"The path element : %s does not exist"` which is perfect.